suspect/processing/channel_combination.py

```diff
@@ -19,3 +24,125 @@ def combine_channels(data, weights=None):
     weighted_data = weights.reshape((len(weights), 1)) * data
     combined_data = weighted_data.sum(axis=0)
     return combined_data
+    
+
+def RAMRS_SVD_weighting(data, phase_correct = True, whiten = False, W = None, noise_range = None):
+    """
+    This channel combination approach operates on raw MRS (spectral) data, 
+    prior to averaging repetitions from multiple acquisitions. It is useful in 
+    cases where there is instability in frequency and phase shifts across 
```

Comment:
My original idea was to do it exactly like you described - but I thought instead it might be a good idea to actually create a function that specifically references this paper, and makes it explicit that it is replicating the method described therein. It becomes sort of self-explanatory in that way, and people can follow along with the paper and see exactly what is going on. So often I'll see an algorithm implemented in a paper, try to find some code that replicates it somewhere, end up cobbling something together from bits and pieces of other libraries, but I'm never quite sure if it's done exactly right. I've worked out the kinks on this approach and tested it so extensively that I thought it would be good to present it in this kind of monolithic way (which it definitely is!) and use the other functions from Suspect internally as needed. An alternative would be to put all of this into an example script somewhere, but then people will have to slog through it, if they find it at all. Anyway, that was my thought process on the whole thing. I'm glad to change it to the method you described, though. I already have a version of that somewhere in my code base.